src/tooltips/tooltips.scss

```diff
@@ -277,3 +277,16 @@
     }
   }
 }
+
+// Responsive tooltips
+//
+// Hide tooltips when primary input mechanism cannot hover (default)
+// Enable tooltips for touch input with .tooltipped-touch
+@media (hover: none) {
+  .tooltipped:not(.tooltipped-touch):hover {
```

Comment:
Is `.tooltipped-touch` a good modifier class name? Or should it be more descriptive `.tooltipped-show-on-touch` `.tooltipped-force-hover`???